econet_hvac_odu.yaml

```diff
@@ -92,6 +92,14 @@ sensor:
     device_class: "pressure"
     state_class: "measurement"
     entity_category: "diagnostic"
+  - platform: econet
```

Comment:
I'll try that as soon as it warms up and I don't risk the family yelling at me for lack of heat.......

Reply:
Using the `econet_hvac_write_float` service, I did write a value to LOCKTIMR, but within seconds the value is overwritten.

  I'm pretty sure this is because the value of the Lockout Timer is generated by the ODU [0x400 in my case] and the write_float service is using a default address [0x340 maybe.  Not sure, haven't dug around enough]

  At any rate, at present this value is read-only

Reply:
Any chance of getting an 'econet_hvac_write_float_to_address' service?